vscode/src/ruby/versionManager.ts

```diff
@@ -56,8 +58,19 @@ export abstract class VersionManager {
   // language server
   abstract activate(): Promise<ActivationResult>;
 
+  runActivatedScript(command: string, options: ExecOptions = {}) {
+    return this.runScript(command, options);
+  }
+
+  buildExecutable(command: string[]): Executable {
+    return {
+      command: command[0],
+      args: command.slice(1),
+    };
+  }
```

Comment:
this might still work
```suggestion
  buildExecutable(command: string, ...args: string[]): Executable {
    return {
      command,
      args,
    };
  }
```